packages/framework/src/Model/GoPay/PaymentMethod/GoPayPaymentMethodFacade.php

```diff
@@ -131,6 +131,7 @@ public function setFromGoPayRawData(
         $goPayPaymentMethodData->imageNormalUrl = $goPayMethodRawData['image']['normal'];
         $goPayPaymentMethodData->imageLargeUrl = $goPayMethodRawData['image']['large'];
         $goPayPaymentMethodData->paymentGroup = $goPayMethodRawData['group'];
+        $goPayPaymentMethodData->available = true;
```

Comment:
is this correct? I expected something like `$goPayMethodRawData['available'];`

Reply:
The thing is, the GoPay API returns only the available payments, so anything returned is available

Reply:
ah, ok, I see :+1: